src/memo.ts

```diff
@@ -229,7 +229,7 @@ export class Memo<T extends MemoType = MemoType> {
    * Returns XDR memo object.
    * @returns {xdr.Memo}
    */
-  toXDRObject(): XDRMemo | null {
+  toXDRObject(): xdr.Memo | null {
```

Comment:
@Akuukis do you know how to deal with this situation in TS? Basically we can't really add typings in `./generated/stellar-xdr_generated'` since it is dynamic and autogenerated. cc @morleyzhi